cmd/relayproxy/api/opentelemetry/otel.go

```diff
@@ -90,9 +91,15 @@ func (o otelErrHandler) Handle(err error) {
 
 var _ otel.ErrorHandler = otelErrHandler(nil)
 
-func initResource(ctx context.Context, serviceName string, version string) (*resource.Resource, error) {
+func initResource(ctx context.Context, serviceName, version string,
+	attribs map[string]string) (*resource.Resource, error) {
+	attrs := make([]attribute.KeyValue, 0, len(attribs))
+	for k, v := range attribs {
+		attrs = append(attrs, attribute.String(k, v))
+	}
```

Comment:
**praise:** This is a good catch, I think `koanf` is messing up things when it is a complex object. This is way better like this, thank you 🙏.